src/app/modules/document-detail/document-edit/document-edit.component.ts

```diff
@@ -253,7 +259,7 @@ export class DocumentEditComponent implements OnInit {
   private setHeaderButtons(document: DocumentDetailDTO) {
     this.headerActions = [
       {
-        label: this.translatedData['GENERAL.CLOSE'],
+        label: this.translateService?.instant('GENERAL.CLOSE'),
```

Comment:
Do not use translateService.instant(). This is not safe. Especially with the way you are loading the translations. headerActions should be an observable which depends on translateService.get()